README.md

```diff
@@ -14,93 +14,23 @@
-
-~~~
-git clone https://github.dev/robotology/gz-sim-yarp-plugins
-mkdir build
-cd build
-cmake -GNinja -DCMAKE_INSTALL_PREFIX=%CONDA_PREFIX%\Library -DCMAKE_PREFIX_PATH=%CONDA_PREFIX%\Library ..
-ninja
-ninja install
-~~~
-
-### Compile from source using apt dependencies on Linux, macOS or Windows
-
-First install some necessary dependencies from apt  
-
-~~~
-sudo apt-get update
-sudo apt-get install lsb-release wget gnupg cmake pkg-config ninja-build build-essential libgtest-dev
-~~~
-
-Then install Gazebo Harmonic:
+```bash
+conda install -c conda-forge gz-sim-yarp-plugins
+```
 
-~~~
-sudo wget https://packages.osrfoundation.org/gazebo.gpg -O /usr/share/keyrings/pkgs-osrf-archive-keyring.gpg
-echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/pkgs-osrf-archive-keyring.gpg] http://packages.osrfoundation.org/gazebo/ubuntu-stable $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/gazebo-stable.list > /dev/null
-sudo apt-get update
-sudo apt-get install gz-harmonic
-~~~
+If you are not familiar with conda or conda-forge, you can read an introduction document in [conda-forge overview](https://github.com/robotology/robotology-superbuild/blob/master/doc/conda-forge.md#conda-forge-overview).
 
-Then, you need to install [`ycm-cmake-modules`](https://github.com/robotology/ycm) and [`yarp`](https://github.com/robotology/yarp), for which no apt binaries are available. You can install them easily via the `robotology-superbuild`, or otherwise with the following commands:
+### robotology-superbuild (advanced)
 
-~~~
-mkdir ~/gsyp_ws
-cd ~/gsyp_ws
-git clone https://github.com/robotology/ycm
-git clone https://github.com/robotology/yarp
-cd ycm
-mkdir build
-cd build
-cmake -GNinja -DCMAKE_INSTALL_PREFIX=~/gsyp_ws/install -DCMAKE_PREFIX_PATH=~/gsyp_ws/install ..
-ninja
-ninja install
-cd ~/gsyp_ws/yarp
-cd yarp
-git checkout v3.9.0
-mkdir build
-cd build
-cmake -GNinja -DCMAKE_INSTALL_PREFIX=~/gsyp_ws/install -DCMAKE_PREFIX_PATH=~/gsyp_ws/install ..
-ninja
-ninja install
-~~~
+You may want to install `gz-sim-yarp-plugins` through the [robotology-superbuild](https://github.com/robotology/robotology-superbuild), an easy way to download, compile and install the robotology software on multiple operating systems, using the [CMake](https://www.cmake.org) build system and its extension [YCM](http://robotology.github.io/ycm). To get `gz-sim-yarp-plugins` when using the `robotology-superbuild`, please enable the `ROBOTOLOGY_USES_GZ_SIM` CMake option of the superbuild.
 
-Then, install `gz-sim-yarp-plugins` itself:
+### Build from source (advanced)
 
-~~~
-git clone https://github.dev/robotology/gz-sim-yarp-plugins
-mkdir build
-cd build
-cmake -GNinja -DCMAKE_INSTALL_PREFIX=~/gsyp_ws/install -DCMAKE_PREFIX_PATH=~/gsyp_ws/install ..
-ninja
-ninja install
-~~~
+If you want to build `gz-sim-yarp-plugins` directly from source, you can check the documentation in [`docs/build-from-source.md`](docs/build-from-source.md).
 
 ## Usage
```

Comment:
The usage part needs to be updated. Specifying the `GZ_SIM_SYSTEM_PLUGIN_PATH` variable manually is only necessary when building from source, in conda and robotology-superbuild this is already handled automatically, I think we need to move this part also in the build from source docs.

Reply:
Oh yes you're right!

Reply:
Done in https://github.com/robotology/gz-sim-yarp-plugins/pull/157/commits/11a014434a1d53f112e333b5c0cf355e07704391